src/PKSim.Infrastructure/Services/ImportObservedDataTask.cs

```diff
@@ -120,19 +123,37 @@ private IReadOnlyList<ColumnInfo> fractionImportConfiguration()
          return columns;
       }
 
-      private IReadOnlyList<ColumnInfo> concentrationImportConfiguration()
+      private IReadOnlyList<ColumnInfo> amountImportConfiguration()
       {
          var columns = new List<ColumnInfo>();
          var timeColumn = createTimeColumn();
+         columns.Add(timeColumn);
 
+         var amountInfo = new ColumnInfo
```

Comment:
Like fraction. I didn't add any error for now, Simple points. We can add that if required later on

Reply:
I think it is required and would add it now.
(also not sure, why we don't have it for fraction)

Reply:
Added for amount. No one ever mentioned it for fraction so clearly it was not missed. Not changing that for now as I am not sure what repercussion this would have